test/test_morse_translator.rb

```diff
@@ -0,0 +1,75 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/morse_translator.rb'
+
+class MorseTranslatorTest < Minitest::Test
```

Comment:
These tests look great!

Reply:
Thanks Megan for the feedback. Have a great night